src/include/Token.h

```diff
@@ -99,4 +99,14 @@ struct Token {
             // TODO: other cases
         }
     }
+    unsigned char getUnOpcode() {
+        switch (type) {
+            case TokenType::Plus:
+                return Opcode::POS;
+            case TokenType::Minus:
+                return Opcode::NEG;
+            case TokenType::Bang:
+                return Opcode::NEG;
```

Comment:
I think this should be `Opcode::NOT`.

Reply:
`NEG` and `POS` does kinda make more sense together than `NOT` and `POS` however 🤔

Reply:
Oh, I meant I thought it was a typo. I’m pretty sure that the opcode that’s supposed to be associated with Bang is NOT. But I haven’t been paying toooo much attention to the VM so I might be wrong 😂